packages/cli-kit/src/public/node/api/app-management.ts

```diff
@@ -25,6 +26,19 @@ async function setupRequest(orgId: string, token: string) {
   }
 }
 
+export const generateFetchAppLogUrlDevDashboard = async (
```

Comment:
instead of `devDashboard` let's use `appManagement`, like other functions in this file